packages/terra-clinical-result/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "terra-clinical-result",
+  "main": "lib/ClincalResult.js",
+  "version": "0.0.0",
+  "description": "A set of components for displaying Clinical Observations like Vital Signs and Laboratory Results.",
```

Comment:
Should this match the read me description of
> The Terra Clinical Result package is a collection of standardized views for presenting clinical results documented to a Patient's Medical Chart, such as Vital Signs, Laboratory Results, and Discretely Documented Values

Reply:
switched here https://github.com/cerner/terra-clinical/pull/574/commits/9eeacc2469da67d5f2d9ad3ce7cc533a407dbf13?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.jsx&file-filters%5B%5D=.scss&file-filters%5B%5D=.snap#diff-f98243208c330dd1438f9a62456b5a8fR5